_posts/2016-02-02-increasing-login-capcity-by-using-node-js.md

```diff
@@ -0,0 +1,72 @@
+---
+layout:     post
+title:      Increasing Login Capacity by Using Node.js
+date:       2016-02-02 20:21:00
+summary:    How we went about transitioning key parts of our stack to Node.js to benefit from the asynchronous nature that it brings.
+author:     tom_lomas
+category:   Product
+tags:       product, lamp, deployment, node, node.js, nginx, uptime, availability
+---
+
+Historically speaking, the sidebar that you may have come to use on Sky Betting & Gaming has been completely powered by a LAMP stack. It has worked well over the years and allows our customers to both login and register, but to also view account history, like open bets and casino spins.
+
+However, as demand increases, especially at crucial times of the year, we need to ensure that our platform remains available and page times are responsive. Login, for example, is used by nearly everyone that comes to the website - whether that is logging in fresh on a desktop browser, opening any of the apps or moving between websites (single sign on) in the Sky Betting & Gaming portfolio.
```

Comment:
Whereas the commas are grammatically correct, the amount of them makes naturally reading this difficult.  May I suggest, "However, as demand increases - especially at crucial times of the year - we need to..."